Packages/com.unity.inputsystem/Documentation~/noisy-controls.md

```diff
@@ -1,15 +1,17 @@
-# Noisy Controls
+# Noisy controls
```

Comment:
Would something like "Handling noisy data from controls" be a better way of describing this?